contents/tutorials/aarrr-how-to-build-pirate-funnel-posthog-with-posthog.md

```diff
@@ -3,7 +3,7 @@ title: Building an AARRR pirate funnel
 sidebar: Docs
 showTitle: true
 featuredImage: ../images/tutorials/banners/aarrr-funnel.png
-featuredTutorial: false
+featuredTutorial: true
```

Comment:
I disagree with swapping this one - this is  a slightly misunderstood version of how to use AARRR. It's not typically seen as a funnel, it's 5 separate metrics, each of which contains submetrics.

The rest of changes make sense to me here!

Reply:
Good to know.

In hindsight, though, I feel like your piece probably shouldn't be in Tutorials. In my head cannon, Tutorials are about how to do stuff in PostHog, and this falls more into the "how to do product analytics" which ought to live on the blog.

At some point I'm going to have another re-org of content and give general product analytics guide content a home.

Reply:
On this, I've put together #3394 to tidy up the [product analytics category](https://posthog.com/blog/categories/product-analytics) on the blog so it just shows broader articles about "how to do analytics" like this tutorial.